validator/node/node.go

```diff
@@ -202,13 +201,14 @@ func (s *ValidatorClient) initializeFromCLI(cliCtx *cli.Context) error {
 			log.Fatalf("Could not get a lock on wallet file. Please check if you have another validator instance running and using the same wallet: %v", err)
 		}
 	}
-
-	dataFlag := flags.WalletDirFlag
+	dataDir := cliCtx.String(flags.WalletDirFlag.Name)
```

Comment:
So this location will never have a db, it is not the root of the accounts directory. Just specifies the general folder to keep
all wallets in prysm, using it as the default in the event an account is not initialized will lead to undefined behavior.

Reply:
https://github.com/prysmaticlabs/prysm/blob/ab76bdad150a5c38e394b213ecf584c30cf887ac/validator/node/node.go#L175 wallet is not defined here

Reply:
wallet is not "" https://github.com/prysmaticlabs/prysm/blob/ab76bdad150a5c38e394b213ecf584c30cf887ac/validator/flags/flags.go#L130

Reply:
@nisdas wdyt?